### ISSUE TITLE
(I think) Remove Required tag from Content property in item.cs

### ISSUE THREAD

**Issue by R4cc** [Bubatz-Inc/Bazaro#11]
Remove the [Required] tag from the 
`public List<MarkdownEntry> Content { get; set; }`
in PropertyBazaro.Data/Models/Item.cs because Items can have no content yet (empty markdown)

(correct me if im wrong)

**Comment by oldviking** [comment]
Thats right, you csn save a empty maekdown note, if we want that. There could only be an title.
